src/main/java/tech/jhipster/lite/module/domain/gradleplugin/JHipsterModuleGradleProfilePlugins.java

```diff
@@ -0,0 +1,101 @@
+package tech.jhipster.lite.module.domain.gradleplugin;
```

Comment:
I'd rather see this one in `javabuildprofile` package:
```suggestion
package tech.jhipster.lite.module.domain.javabuildprofile;
```